packages/block-library/src/navigation/edit/inner-blocks.js

```diff
@@ -98,7 +104,7 @@ export default function NavigationInnerBlocks( {
 			onChange,
 			allowedBlocks: ALLOWED_BLOCKS,
 			prioritizedInserterBlocks: PRIORITIZED_INSERTER_BLOCKS,
-			__experimentalDefaultBlock: DEFAULT_BLOCK,
+			__experimentalDefaultBlock: navDefaultBlock,
 			__experimentalDirectInsert: shouldDirectInsert,
```

Comment:
In terms of stabilizing `__experimentalDefaultBlock`, I think it could be worth looking at stabilizing `__experimentalDirectInsert` too, as I believe the two props were designed to work together.

Perhaps if there's consensus that `defaultBlock` should be added as a block attribute, the same should also apply to `directInsert`. 🤔

Reply:
Resolved by https://github.com/WordPress/gutenberg/pull/52083